types/bunyan-format/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for bunyan-format 0.2
+// Project: https://github.com/thlorenz/bunyan-format
+// Definitions by: Piotr Roszatycki <https://github.com/dex4er>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
Add a `// TypeScript Version: 2.1` comment on this line (just like `bunyan` has)